transactions/rbf.ts

```diff
@@ -274,16 +274,20 @@ class RbfTransaction {
     return this._paymentUtxos;
   };
 
-  private getBip32Master = async () => {
+  private getBip32Master = async (options: CompileOptions) => {
     // keep this method short so seed phrase is as short lived as possible
-    const seedPhrase = await this.options.getSeedPhrase();
+    const seedPhrase = await options.getSeedPhrase();
     const seed = await bip39.mnemonicToSeed(seedPhrase);
     return bip32.fromSeed(seed);
   };
 
-  private signTxSoftware = async (transaction: btc.Transaction): Promise<btc.Transaction> => {
+  private signTxSoftware = async (transaction: btc.Transaction, options?: CompileOptions): Promise<btc.Transaction> => {
+    if (!options?.getSeedPhrase) {
+      throw new Error('Options are required for non-dummy transactions');
+    }
```

Comment:
question: should we change the message to "getSeedPhrase option is required" here? since it's the single prop the code is validating